presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraType.java

```diff
@@ -220,6 +223,7 @@ public static NullableValue getColumnValue(Row row, int position, CassandraType
                 case VARINT:
                     return NullableValue.of(nativeType, utf8Slice(row.getVarint(position).toString()));
                 case BLOB:
+                case DURATION:
```

Comment:
Any reason why DURATION is not add to getColumnValueForCql() and objectToString()?

Reply:
just added,